lib/draw.c

```diff
@@ -110,6 +113,90 @@ mesh importMeshFromOBJFile (char * pathToFile)
     return newMesh;
 }
 
+int importJSON(const char *file_path, data *importData_struct)
```

Comment:
is this struct called data from cJson?
If not, maybe rename to something more descriptive

Reply:
i renamed it to renderConfig